groups/tests/test_models.py

```diff
@@ -124,6 +124,60 @@ def test_is_deleted(self):
         comment.delete_state()
         self.assertTrue(comment.is_deleted())
 
+    def test_subscribers_discussion(self):
+        """The users subscribed to the discussion are included."""
+        comment = factories.BaseCommentFactory.create()
+        subscriber, other = factories.UserFactory.create_batch(2)
+        comment.discussion.subscribers.add(subscriber)
```

Comment:
Would be good to see some of the cloned code in these five tests moved into a `setUp()`.